dir.props

```diff
@@ -10,7 +10,7 @@
 
   <!-- Build Tools Versions -->
   <PropertyGroup>
-    <BuildToolsVersion>1.0.25-prerelease-00131</BuildToolsVersion>
+    <BuildToolsVersion>1.0.25-prerelease-00132</BuildToolsVersion>
```

Comment:
I just built 00133 and will be submitting a PR for it shortly.  Do you want to just pick up 133 in your change?